tsconfig.json

```
@@ -11,6 +11,7 @@
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
+    "noFallthroughCasesInSwitch": true,
```

This might cause issues elsewhere because I recall seeing a fall through being used somewhere. Could be wrong though...It certainly might catch someone off guard? Dunno...

---

afiact a violation of `noFallthroughCasesInSwitch` will result in build breaking and not a silent error.